modules/ROOT/pages/_partials/symbolicate-iOS.adoc

```diff
@@ -0,0 +1,80 @@
+== Symbolication
+
+In order to make sense of a crash report you will need to symbolicate
+ its content.
+Symbolication being the process of replacing memory addresses in a crash report with human-readable function names and line numbers.
+To do this you need to have debug symbol (`.dSYM`) files for the application and for Couchbase Lite's framework.
+
+== Finding local dSYM
+
+To locate the local dSYM files:
+
+* For scheme based builds, they are located in the derived data path
+* For the target based builds, they are located in the configuration build directory path, as specified in the _target_ setting
+* For archives, they are located in `.xcarchive`.
+
+TIP: If you still cannot locate the file, then use `mdfind` as described in the
+_Symbolication Troubleshooting_ section of the _Apple Technical Note TN2151 --
+Understanding and Analyzing Application Crash Reports_: https://developer.apple.com/library/archive/technotes/tn2151/_index.html
+
+== The CouchbaseLite dSYM
+
+Couchbase Lite's `dSYM` folder is located in the same directory as the Couchbase Lite binary framework (this also applies for _Cocoapod_ and _Carthage_).
+It has the same filename as framework but has a `.dSYM` extension.
+
+.dSYM in CouchbaseLiteSwift
+====
+----
+couchbase-lite-swift_enterprise_2.7.0/iOS/CouchbaseLiteSwift.framework.dSYM
+----
+====
+
+== Bitcode enabled Apps
+
+Apple creates new dSYM files for apps uploaded to the _App Store_ or _TestFlight_ with _Bitcode_ enabled.
+To symbolicate crash reports generated by these apps you must download the generated dSYM files. Do this using _Xcode's Organizer_ or direcly from _App Store Connect_.
+
+=== Downloading dSYM using Xcode Organizer
+
+. Within _Xcode's_  _Archives Organiser_, select the required archive
+. Within the _Archive Information_ dialog panel, select *Download Debug Symbols* button
++
+The dSYM files are downloaded to the archive bundle in a folder named `dSYMs`
++
+See _Download debug symbols_: https://help.apple.com/xcode/mac/current/#/devef5928039
+
+=== Downloading dSYM directly from App Store Connect
+
+. Inside _My Apps_ choose the *Activity* section
+. Locate the required build (the one you want to download dSYM files for)
+. In the _Includes Symbols_ column, select the download `dSYM` link .
+
+== Using third-party crash report tools
+
+You can upload your dSYM files to third-party crash reporting tools for analysis, either automatically via script or a manual _zip_ file upload. Links for some of the more popular crash reporting tools are included below.
+
+.Third party crash reporting tools
+====
+* Crashlytics legacy: https://docs.fabric.io/apple/crashlytics/missing-dsyms.html
```

Comment:
As Priya suggested, we should include new Crashlytics link with legacy. 
https://firebase.google.com/docs/crashlytics/get-deobfuscated-reports

Reply:
Thanks for the timely review @jayahariv.

Since the legacy site ends in May I have replaced the original link with the Firebase Crashlytics link; see commit [9c2ee5291e27a4f1cc98af2a8749c496fb19244f](https://github.com/couchbase/docs-couchbase-lite/pull/203/commits/9c2ee5291e27a4f1cc98af2a8749c496fb19244f).

Is this good to go?